home-manager/home-manager

```diff
@@ -45,11 +45,7 @@ function setWorkDir() {
 
 # Checks whether the 'flakes' and 'nix-command' Nix options are enabled.
```

Comment:
Should update this, too, now that we aren't just checking for the presence of those options.

Reply:
Great catch, I updated the comment. What do you think?